src/docs/src/intro/security.rst

```diff
@@ -312,6 +312,64 @@ several *mandatory* fields, that CouchDB needs for authentication:
 Additionally, you may specify any custom fields that relate to the target
 user.
 
+Password Schemes
+----------------
+
+CouchDB supports several password hashing schemes:
+
+Simple
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think, that the lines below and above a header must match the length of the text.